### ISSUE TITLE
Add code coverage info

### ISSUE THREAD

**Issue by junghuk** [inf295uci-2015/SeeSoftOnD3#4]
Show code coverage info 

Features:
- show coverage info when it is zoomed.

**Comment by VijayKrishna** [comment]
We probably need an Execution Data API (and its implementation) to read in the coverage data into the visualization. I was also thinking that we should probably compute the suspiciousness values based on the code coverage and only expose that in this Execution Data API. Thoughts?

**Comment by junghuk** [comment]
I agree with you. Tacoco.json is too big to handle in JS. We only need suspiciousness values for each file or line numbers in file.
